index.bs

```diff
@@ -2160,7 +2225,19 @@ Constructors</h4>
 			2. Set the {{[[rendering thread state]]}} for
 				<var>c</var> to <code>"suspended"</code>.
 
-			3. Construct an {{AudioDestinationNode}} with its
+			3. Determine the {{[[render quantum size]]}} for this
+				 {{OfflineAudioContext}}, based on the value of
+				 the {{OfflineAudioContextOptions/renderSizeHint}}:
+
+				 1. If it has the default value of <code>"default"</code> or
+					<code>"hardware</code>", set the {{[[render quantum size]]}} private
+					slot to 128.
+
+				 2. Else, if an integer has been passed, the User-Agent can decide to
```

Comment:
Don't we need the third step where the UA decides its own value because (2) failed?